t/ui/14-dashboard.t

```diff
@@ -53,7 +53,7 @@ is(scalar @{$driver->find_elements('h4', 'css')}, 2, 'only one build per group s
 
 $driver->find_element('opensuse', 'link_text')->click();
 
-is(scalar @{$driver->find_elements('h4', 'css')}, 3, 'default number of builds shown');
+is(scalar @{$driver->find_elements('h4', 'css')}, 4, 'default number of builds shown');
```

Comment:
The default number is 10 (see lib/OpenQA/WebAPI/Controller/Main.pm, line 146) so the increase is ok (the default sets the _maximum_ number of builds).